myhpom/models/cloudfactory.py

```diff
@@ -0,0 +1,146 @@
+
+import json
+import datetime
+from django.db import models
+
+
+class CloudFactoryRun(models.Model):
+    """Store information about current and past CloudFactory runs."""
+
+    line_id = models.CharField(
```

Comment:
It'd be helpful to have a foreign key reference to the AD that this model is related to (with on_delete=SET_NULL maybe?). That'd be helpful to see in the admin, and helpful maybe for obtaining a reference to the AD and user when constructing the request payload...

Reply:
Yes, this is the main shortcoming of my approach of generalizing the CloudFactory models. Please have a look at what I've done for the admin in MH-274 before you pass final judgment on that approach.